_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,343 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
```

Comment:
### Gitter chatroom for Neovim

In addition to the mailinglist and the IRC channel, Neovim now has a [chatroom on Gitter.im][gitter-room]. Besides using the interface, you can also [use your IRC client](irc.gitter.im) to connect to it.

-- ... or something like that :grin: